frame-metadata/src/v15.rs

```diff
@@ -171,10 +171,12 @@ impl IntoPortable for RuntimeApiMethodParamMetadata {
 	serde(bound(serialize = "T::Type: Serialize, T::String: Serialize"))
 )]
 pub struct ExtrinsicMetadata<T: Form = MetaForm> {
-	/// The type of the extrinsic.
-	pub ty: T::Type,
 	/// Extrinsic version.
 	pub version: u8,
+	/// The prefix of the extrinsic.
+	///
+	/// This is the type of the encoded length of this extrinsic.
+	pub len_ty: T::Type,
```

Comment:
Remember that it makes no real difference whether it's a `Compact<u32>` or a `Compact<u128>`; both encode and decode the same way (the only thing that we learn is that the max size of the type is, but I guess I wouldn't expect an extrinsic to exist that was big enough that this would evr matter :)

As I said on the other PR, I'm not yet sure if this is actually useful in any way or if it's just something that we can assume is a detail of how version 4 extrinsics are encoded :)